### ISSUE TITLE
Fine tuning on different dataset

### ISSUE THREAD

**Issue by Soumyabrata** [hellochick/PSPNet-tensorflow#29]
Hi,
Is it possible to fine-tune the saved model files (on cityscape or ade20k dataset), on a completely different dataset? I'm interested to train on a different dataset, which consists of 2 labels. Of course, training from scratch on this new dataset is not a good idea, because the batch size can be set to a maximum value of 2, in a normal GPU setting. 
It is obvious that we get a shape mismatch error, when we use model files from cityscape dataset (containing 19 labels) on my new dataset (with 2 labels).

``InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [1,1,512,2] rhs shape= [1,1,512,19]``

Any possible ideas or suggestions?

**Comment by hellochick** [comment]
Hey, you can take a look at this issues #14.
